### PR TITLE
Review Pull Request

### PR DESCRIPTION
**Pull request by tamoorsaeed22**
This is a code review project

### CODE REVIEW

**Review by tamoorsaeed22 — COMMENTED** [review]
kindly add the MIT license key and  a descriptive title different from the document. This will help the user to have a clear idea about your project.

File: LICENCE [hunk 1]
```diff
@@ -1,21 +0,0 @@
-MIT License
-
-Copyright (c) 2023 Tamoor Saeed
```
Comment (tamoorsaeed22) [inline LICENCE:3]: Please add License MIT Key to the Review branch a LICENSE key is a unique identifier that is often used in software licensing to activate or verify a user's license for a particular software product. The key typically consists of a series of alphanumeric characters that are provided to the user upon purchase of the software license.

Comment (AlphaNtihinduka) [reply LICENCE:3]: - [x] You have so far done well to use the inline comments 👏 👍 . However, you should try as much as possible to ensure that the review comment is more detailed. You should state the request needed in a polite way, explain to why it is necessary, give more details on exact part required. In addition, you may also attach the external resources you think can help the contributor(s) to apply the changes you have required easily ☺️

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @tamoorsaeed22 🙋‍♂️
Good job working on using the inline comment.

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is not yet professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- [ ] I suggest that you to provide atleast two comments review. This will help you have a good practical understanding of how the code review is done. Additionally it will help you to make further practice and be able to identify the bugs in the code 👍 .

## Optional suggestions
- [ ] [OPTIONAL]  


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Comment by tamoorsaeed22** [comment]
@AlphaNtihinduka A detailed MIT advice has been updated so the person can get the idea.

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @tamoorsaeed22  👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far 👍 
 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight 👍 
✔ Good review 👍 

## Required Changes ♻️
- [x] **Kindly as suggested by the previous reviewer** I think it would be better also to add at least one more review comment to gain more experience in reviewing projects 😎😎

Kindly check this [video](https://www.youtube.com/watch?v=lSnbOtw4izI) for more guidance on giving github inline comments 👍 

## Optional request 
- [ ] **[Optional]** I think It would be better to use a couple of emojis or gifs in the review to make the code review more human-friendly 👽😍

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tamoorsaeed22** [comment]
@TuyishimireEric another comment have been added as required two comments .

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @tamoorsaeed22 :wave:, 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Clear & empathetic communication :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Good use of in-line comment :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
- As against what your claim that you have added another in-line comment, I can't find any.
Since it is a requirement to have at least 2 inline-comments:
- [x] Kindly as suggested by the previous reviewer I think it would be better also to add at least one more review comment to gain more experience in reviewing projects :wink: 
- Kindly check this [video](https://www.youtube.com/watch?v=lSnbOtw4izI) for more guidance on giving github inline comments :+1: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- I think It would be better to use a couple of emojis or gifs in the review to make the code review more human-friendly :vulcan_salute: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @tamoorsaeed22 , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [x] Good work using inline comments for your first comment ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._
- [x] As previous reviewers have stated, You need to have at least 2 reviews but there is only one. Please follow the video link provided by previous reviewers and add another comment. Please ensure that your comment meets the following:-
 - The comment should be phrased politely as a suggestion instead of an order. For example "**think it would be better to use semantic tags**" instead of "**Use semantic tags**"
 - Include an explanation, a "**why**", for each requested change
 - The comment should be clear and easy to understand
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tamoorsaeed22** [comment]
@wayungi As you can see there are almost too many comments from my side for updating I don't know why there are too many comments so I would appreciate it if you can see and scroll up I have left almost 4,5 comments review in line

**Comment by AbayomiOlaoye** [comment]
@tamoorsaeed22 :+1: Sorry about that. The comment isn't just comments like the ones you are having with us as reviewers. It is the one you give to the student whose project you are reviewing - which in this case is you. So, the comments are not your conversation with code reviewers but with the student, and the only comment you gave the student was [this one](https://github.com/tamoorsaeed22/Review-Project-Trials/pull/1#discussion_r1122746429). So, kindly add at least one more of that polite comment :pray:

**Comment by tamoorsaeed22** [comment]
### Highlights 👍

- Good github workflow
### Required changes ♻️

- Checkout the comments for more information.
Kindly add a README file to your project to make it look more professional and this makes it easier for other people to understand your project and objectives. Check out this [template](https://github.com/microverseinc/readme-template).

-  In addition, you may also attach the external resources you think can help the contributor(s) to apply the changes you have required easily ☺️

- Additionally it will help you to make further practice and be able to identify the bugs in the code 👍 .

**Review by AbelG101 — APPROVED** [review]
# Hi @tamoorsaeed22 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- [ ] You can consider preceding each comment with a `- [ ]` to create a checkbox as this will help your partner ensure they're on track with each change you have requested by ticking off the implemented changes. 👍

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**